x-pack/plugins/security_solution/public/detections/components/alerts_table/default_config.tsx

```diff
@@ -192,7 +192,7 @@ export const getAlertsDefaultModel = (license?: LicenseService): SubsetDataTable
 
 export const getAlertsPreviewDefaultModel = (license?: LicenseService): SubsetDataTableModel => ({
   ...getAlertsDefaultModel(license),
-  columns: getColumns(license),
+  columns: getRulePreviewColumns(license),
```

Comment:
While working on this issue, I found that we accidentally set incorrect columns to rule's preview table. The appropriate changes were done here https://github.com/elastic/kibana/pull/139478/files#diff-2676630f0684f0ba6f1d3e42850d059ded14bc72da00bf8e925bcc4ece71a87bR165